actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -284,17 +308,77 @@ workflows:
                     destination_user: "funk_901"
                     destination_path: <% $.irma_remote_path %>
                     rsync_include_file: /etc/arteria/misc/hiseq.rsync
-                    md5_output_file: "ngi.md5"
+                    md5_output_file: "checksums.md5"
                 on-success:
-                  - mark_as_finished
-                  - notify_finished
+                  - irma_check_md5sums
 
-            # TODO Need to add checksum verification on irma
-            # TODO Need to add starting the ngi pipeline on irma
+            irma_check_md5sums:
+               action: core.http
+               input:
+                    method: "POST"
+                    url: <% $.irma_checksum_base_url %>/start/<% $.runfolder_name %>?apikey=<% $.irma_api_key %>
+                    body: '{"path_to_md5_sum_file": "MD5/checksums.md5"}'
+                    verify_ssl_cert: False
```

Comment:
Stackstorm doesn't support (at the moment at least) verifying against other CA bundles than the one supplied by Mozilla. This includes a DigiCert root certificate that can be used for verifying TERENA certs. But I don't think UPPMAX would want us to apply for a certificate under an UPPMAX domain name... so that would require us to depend on them taking care of this. Which I have sort of ignored for now..